lib/jekyll.rb

```diff
@@ -160,11 +160,11 @@ def sanitized_path(base_directory, questionable_path)
 
       questionable_path.insert(0, "/") if questionable_path.start_with?("~")
       clean_path = File.expand_path(questionable_path, "/")
-      clean_path.sub!(%r!\A\w:/!, "/")
 
-      if clean_path.start_with?(base_directory.sub(%r!\A\w:/!, "/"))
+      if clean_path.start_with?(base_directory)
         clean_path
       else
+        clean_path.sub!(%r!\A\w:/!, "/")
```

Comment:
> I'd have to test this because I'm not quite sure but I have a feeling...

@envygeeks : Did you get the time to test if this had any side-effects..?